### ISSUE TITLE
Invalid content in xml

### ISSUE THREAD

**Issue by marcuzy** [mk-j/PHP_XLSXWriter#62]
MS Excel open xlsx files with errors.

``` xml
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
<logFileName>
error018240_01.xml
</logFileName>
<summary>
Обнаружены ошибки в файле "G:\unload.xlsx"
</summary>
<removedRecords summary="Список удаленных записей:">
<removedRecord>
Removed records: Formula from part /xl/worksheets/sheet1.xml
</removedRecord>
</removedRecords>
</recoveryLog>
```

I assume this problem relates with bad chars (http://stackoverflow.com/questions/730133/invalid-characters-in-xml). XLSXWriter::xmlspecialchars contain simple control, I think that this is a serious problem for some cases.

**Comment by marcuzy** [comment]
This problem relates with cell type autodetecting in XLSXWriter::writeCell. 
```php
//...
} elseif (is_string($value) && $value{0}=='='){
//...
```
In my case some values contains "=" char in beginning of string. Excel supports values with "=" if change cell type manually to string, otherwise it's interpreting as a formula. XLSXWriter::writeCell doesn't support mandatory typing in this case.

Bad characters still are the actual topic. I think that is lib's task.

**Comment by prafful-panwar** [comment]
is there any easy way to remove all these character from entire array that we are passing in $writer->writeSheet($exceldata, 'Sheet1', $Newheader); function ?

**Comment by mk-j** [comment]
This should be fixed in the latest version 0.30
